### PR TITLE
BETA: Register fluid.is-a.dev

### PR DESCRIPTION
**Pull request by ann-dvlpr**
Added `fluid.is-a.dev` using the [dashboard](https://manage.is-a.dev).

### CODE REVIEW

**Comment by fluid-developer** [comment]
im not sure if i did right record type as im very very new to this, all i need is to just make my github pages be linked with fluid.is-a.dev as Github supports that

File: domains/fluid.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "fluid-developer",
+        "email": "ne0xch4r4@gmail.com"
+    },
+    "record": {
+        "TXT": "test"
```
Comment (CuteDog5695) [inline domains/fluid.json:7]: TXT doesnt even do anything

**Review by iostpa — DISMISSED** [review]

**Comment by CuteDog5695** [comment]
> im not sure if i did right record type as im very very new to this, all i need is to just make my github pages be linked with fluid.is-a.dev as Github supports that

if you want to use Github Pages follow along this document https://www.is-a.dev/docs/github_pages/

**Comment by fluid-developer** [comment]
okay, i followed the guide you sent. you can try checking it again

**Review by wdhdev — APPROVED** [review]

**Pull request merged**